docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ax.bar_label(ax.containers[0])
</pre><p>Do you need to call <code>plt.show()</code> or just end this cell with a <code>;</code> to avoid showing the <code>[Text(0, 0, '26), ...]</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Done